runtime-decompiler/src/main/java/org/jrd/backend/data/Directories.java

```diff
@@ -50,4 +53,17 @@ public static String getXdgJrdBaseDir() {
         return xdgConfigHome;
     }
 
+    public static String getJrdLocation(){
+        if(System.getProperty("jrd.location") == null){
+            OutputController.getLogger().log(OutputController.Level.MESSAGE_DEBUG, "jrd.location environment variable not found, using fallback");
+            return Paths.get(".").normalize().toAbsolutePath().toString();
```

Comment:
this is wrong. this is again CWD

Reply:
ye this is the backup that you mentioned in [https://github.com/pmikova/java-runtime-decompiler/pull/53#discussion_r309260890](here)